### PR TITLE
Added sponsorship back in

### PR DESCRIPTION
**Pull request by lindaicing**
Screen Shot 2020-07-04 at 1 29 19 AM

On Hover
Screen Shot 2020-07-04 at 2 21 45 AM

### CODE REVIEW

**Comment by lindaicing** [comment]
Setup Popup for Sponsor Format
Screen Shot 2020-08-25 at 1 06 48 AM

File: Extension/popup.html [hunk 1]
```diff
@@ -107,11 +107,14 @@ <h6 id="leapingbunny" class="tooltip">
...
+                    <div id="sponsorGroup">
                         <a href="#" id="sponsoredImg">
-                            <p class="sponsorRating"> 9.5</p>
+                            <p class="sponsorRating">9.5</p>
                             <img src="https://images.unsplash.com/photo-1490885578174-acda8905c2c6?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2500&q=80" alt="Sponsored Image" id="sponsoredImg">
-                            <p id="sponsoredBy">Sponsoring Company</p>
+                            <p id="sponsoredBy">
+                                <span id="sponsorCompany">Sponsoring Company</span>
+                                <span id="sponsorPrice">$00.00</span>
+                            </p>
```
Comment (CubeOfCheese) [inline Extension/popup.html]: Looks good! Now the next step is to remove all the text and that image and instead render it with javascript
ie .textContent and .src

File: Extension/popup.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ function loadExtension(ethicliStats) {
       adjustSubscores();
     }
     function adjustSubscores() {
-        var fullheight = 360;
+        var fullheight = 680;
```
Comment (CubeOfCheese) [inline Extension/popup.js:17]: Does this mean it will display as 680 even when there isn't an ad?

**Comment by CubeOfCheese** [comment]
I don't see anything wrong per se with this PR, but it's not at the point where it should be merged in because it's not quite the complete feature

File: Extension/popup.js (hunk 2 above)
Comment (lindaicing) [reply Extension/popup.js:17]: Yeah I thought the "conditional rendering" bit could be in another PR to reduce confusion, esp since no data is linked yet

**Comment by lindaicing** [comment]
> I don't see anything wrong per se with this PR, but it's not at the point where it should be merged in because it's not quite the complete feature

I thought it might be good to merge it and start linking data (in another PR) and this would just set up the barebones infrastructure for that

**Comment by lindaicing** [comment]
Update: 
Screen Shot 2020-09-20 at 11 46 43 AM
Screen Shot 2020-09-20 at 11 46 56 AM

**Review by CubeOfCheese — DISMISSED** [review]
This a big boy. Good job Linda!

**Review by CubeOfCheese — APPROVED** [review]

**Pull request merged**